tox.ini

```diff
@@ -1,8 +1,14 @@
 [tox]
-envlist = py27,py34,py35,py36,pypy
+envlist = flake8,py27,py34,py35,py36,pypy
 
 [testenv]
 commands =
     zope-testrunner --test-path=src []
 deps =
     .[test]
+
+[testenv:flake8]
+basepython = python3.6
+skip_install = true
+deps = flake8
+commands = flake8 --doctests src setup.py
```

Comment:
I recall @mgedmin had a comment about arguments for flake8 being in `setup.cfg`, not `tox.ini`, but I don't recall how that got resolved.

Reply:
Hm, as @mgedmin did see this PR but did not comment on this, I think I am free to keep it.

Reply:
So the reason why I prefer flake8 configuration to be in a setup.cfg instead of specified on the command line is that people have text editors and IDEs with flake8 integration, and those plugins will never ever think to parse a tox.ini for possible command-line options applicable to a codebase.  Also, duplicating the command-line options in a tox.ini and a .travis.yml is not fun.

(It's also why I prefer .coveragerc to command-line options passed to coverage run.)